### PR TITLE
refactor(combobox): tailwindify

### PR DESCRIPTION
**Pull request by julio8a**
Supports #1500 

@paulcpederson will work on the 24, 32, 48 sizing after this merge. 

the main change here is the border color, input-border-color is used and best for accessibility

### CODE REVIEW

**Comment by julio8a** [comment]
@bstifle ^ 

Please review screener

File: src/components/calcite-combobox/calcite-combobox.scss [hunk 1]
```diff
@@ -137,21 +138,19 @@
 }
 
 .list-container {
-  overflow-y: auto;
-  max-height: 100vh;
+  @apply overflow-y-auto
+    max-h-screen
+    bg-foreground-1;
   width: var(--calcite-dropdown-width);
   background: var(--calcite-ui-foreground-1);
```
Comment (paulcpederson) [inline src/components/calcite-combobox/calcite-combobox.scss:145]: you have to delete this line here...

Comment (julio8a) [reply src/components/calcite-combobox/calcite-combobox.scss]: darn it, forgot to delete it after tailwindyfing it! DONE!

**Comment by julio8a** [comment]
This one is good to go

**Comment by paulcpederson** [comment]
@julio8a you need to run the code locally and look at the demo page. Single select is showing both placeholder and value.

Screen Shot 2021-03-22 at 9 32 33 AM
Screen Shot 2021-03-22 at 9 32 24 AM

**Comment by julio8a** [comment]
thx @paulcpederson just pushed an update. verified locally.

File: src/components/calcite-combobox/calcite-combobox.scss [hunk 2]
```diff
@@ -32,46 +31,48 @@
...
   font-size: inherit;
-  font-family: inherit;
-  padding: 0;
-  background-color: transparent;
-  border: none;
-  color: var(--calcite-ui-text-1);
-  appearance: none;
   height: var(--calcite-combobox-input-height);
   line-height: var(--calcite-combobox-input-height);
   min-width: 120px;
```
Comment (jrowlingson) [inline src/components/calcite-combobox/calcite-combobox.scss:67]: `theme("spacing.32")`?

**Review by jrowlingson — APPROVED** [review]

**Pull request merged**